src/views/properties/properties.jsx

```diff
@@ -186,6 +193,28 @@ export const Properties = () => {
           />
         </div>
       </div>
+      {showArchiveModal && 
+        <Modal
+          content={
+            <div className="content is-small">
+              <p>You have selected the following {selectedProperties.length} properties to be archived:</p>
+              <ul className="archive-properties-list">
+              {selectedProperties.map(p => (
+                <li>{p.name}</li>
+              ))}
+              </ul>
+              <br/>
+              <p>Are you sure you want to archive these properties?</p>
+            </div>
```

Comment:
This works great, but I think maybe we should use a different font size and/or weight for some of this text.  It seems a little out of place to have it all the same small text when there are multiple lines:

![image](https://user-images.githubusercontent.com/44326005/111409133-9ac75000-8693-11eb-99fb-56aea8e268ea.png)